src/visualization_functions.cpp

```diff
@@ -269,6 +269,41 @@ void PathVisualization(const std::vector<Eigen::Vector3d> &total_path,
     markers->markers.push_back(line_list);
 }
 
+void PathVisualization(const std::vector<Eigen::Vector3d> &total_path,
+                       const std_msgs::ColorRGBA &color,
+                       const std::string &inertial_frame_id,
+                       const std::string &ns,  // namespace
```

Comment:
rather than explain what the variable name means using a comment, we could just call the variable `namespace`

Reply:
hahaha we can't name it `namespace`, it is a reserved name for c++